pkg/config/config.go

```diff
@@ -449,3 +450,19 @@ func (c *Config) GetUIEnabled() bool {
 func (c *Config) GetLoginDuration() time.Duration {
 	return c.values.Auth.LoginDuration
 }
+
+func (c *Config) GravelerRepositoryCacheConfig() ref.CacheConfig {
```

Comment:
```suggestion
func (c *Config) GetGravelerRepositoryCacheParams() ref.CacheConfig {
```
for consistency, e.g. like GetBlockAdapterS3Params.  Also below?

Reply:
Thanks - I'll address the new/old practice in a different pr